_libs/nav/head.js

```diff
@@ -6,116 +6,203 @@ const navItems = [
     href: "/info/data",
     sections: [
       {
-        name: "Choosing a model",
-        href: "/getting-started/choosing-a-model/",
+        name: "Loading and Accessing Data",
+        href: "/data/loading/",
         tags: ["Data Processing"],
+        ilos: [
+            "Understand how to load and access various datasets in R using RDatasets.jl",
+            "Learn how to save and load a local dataset in CSV format using CSV.jl"
+        ]
       },
```

Comment:
```suggestion
        name: "Ingesting Data and Choosing a Model",
        href: "/data/loading/",
        tags: ["Data Processing", "Model Search"],
        ilos: [
            "Understand how to load and access various datasets in R using RDatasets.jl",
            "Learn how to discover models matching your data.",
            "Learn how to save and load a local dataset in CSV format using CSV.jl"
        ]
      },
```

Small additional suggestion here, I prefer tags that are no captilalized, unless we have proper names or acronyms, i.e., "regression" not "Regression", and "data processing" not "Data Processing". This is consistent with the tags on the ModelDescriptors.toml file (and LearnAPI.jl) right?

Reply:
The tags in the model browser are capitalized. Model `x` is said to belong to tag `y` if it occurs under the "headline" `y` in the model browser generated by the ModelDescriptors.toml (MLJ website only looks at the model browser and generates it from ModelDescriptor.toml)

For instance all the following models will be under the tag "Regression":
<img width="938" alt="image" src="https://github.com/JuliaAI/DataScienceTutorials.jl/assets/49572294/edf4e898-e9be-44a9-9c28-56869ba0be67">

Reply:
Resolved in offline discussion. We keep capitlized string tags with spaces, to match exactly how they will appear in headings.